index.html

```diff
@@ -25,20 +25,25 @@
         <figure class="btn-scale nav-icon">
           <img src="./assets/images/Union.svg" alt="nav icon" />
         </figure>
-        <ul class="nav-items">
-          <li><a href="#Portfolio"> Portfolio </a></li>
+        <div class="close-navbar">
+          <figure class="btn-scale close">
+            <img src="./assets/images/Cancel.png" alt="nav icon" />
+          </figure>
+          <ul class="nav-items">
+            <li class="nav-list"><a href="#Portfolio"> Portfolio </a></li>
 
-          <li><a href="#About"> About </a></li>
+            <li class="nav-list"><a href="#About"> About </a></li>
 
-          <li><a href="#Contact"> Contact</a></li>
+            <li class="nav-list"><a href="#Contact"> Contact</a></li>
 
-          <li>
-            <a href="#Contact">
-              <figure class="btn-scale">
-                <img src="./assets/images/Mail.svg" alt="nav icon" /></figure
-            ></a>
-          </li>
-        </ul>
+            <li class="nav-list">
+              <a href="#Contact">
+                <figure class="btn-scale">
+                  <img src="./assets/images/Mail.svg" alt="nav icon" /></figure
+              ></a>
+            </li>
+          </ul>
```

Comment:
- [ ] On your mobile menu kindly consider sticking to the Figma design as much as possible to make your design look exactly like the one given in Figma check the images below to see where to fix
- Your design 👇 
<img width="379" alt="Screen Shot 2023-06-20 at 11 34 34" src="https://github.com/Errytagedesign/microverse-portfolio/assets/60146030/3d1c42f0-dd2d-4863-98e1-10346c338cc5">

- Figma design 
<img width="366" alt="Screen Shot 2023-06-20 at 11 36 50" src="https://github.com/Errytagedesign/microverse-portfolio/assets/60146030/fe70b261-970b-4d1b-aa81-84c3befc4e83">

check also on the background color 😄